app/docs/statik/init.go

```diff
@@ -0,0 +1,3 @@
+package statik
+
```

Comment:
lol statik.go file has 1.6MB

Reply:
yeah, it embed the whole `app/docs/swagger-ui` directory.